x-pack/test/reporting_api_integration/reporting_and_security/network_policy.ts

```diff
@@ -20,7 +20,7 @@ export default function ({ getService }: FtrProviderContext) {
    * The Reporting API Functional Test config implements a network policy that
    * is designed to disallow the following Canvas worksheet
    */
-  describe('reporting network policy', () => {
+  describe('Network Policy', () => {
```

Comment:
The name of the test was `Reporting reporting Network Policy` and removes the repetition

Reply:
ah, good to know